test/client/getFee/index.ts

```diff
@@ -32,6 +33,7 @@ export default <TestSuite>{
     // (fee will actually be 51539.607552)
     client._maxFeeXRP = '51540'
     client.connection.request({
+      // @ts-ignore TODO: resolve
```

Comment:
What's the trigger to resolve this? (Just to be sure we don't forget)

Reply:
It'll be resolved when we rewrite the mocking infrastructure for these tests.